common/src/main/java/com/wynntils/wynn/model/WorldStateManager.java

```diff
@@ -142,21 +144,22 @@ public static void onTabListFooter(PlayerInfoFooterChangedEvent e) {
 
         if (!footer.isEmpty()) {
             if (HUB_NAME.matcher(footer).find()) {
-                setState(State.HUB, "", null);
+                setState(State.HUB, "");
             }
         }
     }
 
     @SubscribeEvent
     public static void update(PlayerDisplayNameChangeEvent e) {
         if (!onServer()) return;
+        if (!e.getId().equals(WORLD_NAME_UUID)) return;
```

Comment:
UUIDs again? What is this used for?

Reply:
Slightly quicker return in case it's not about world changes, and slightly higher accuracy. I used to pick this up dynamically, but it never changes, and iirc @HeyZeer0 confirmed that it is always the same.

Reply:
I hope this won't bite us again. But let's go with it.